controllers/promotions/deleter.js

```diff
@@ -0,0 +1,36 @@
+const { promotion } = require('../../databases/db');
+const responses = require('../../utils/responses/responses');
+const { fileController } = require('../filesControllers');
+const getImagePath = require('../../utils/helpers/get-image-path');
+
+/**
+ * It deletes a promotion from the database and deletes the image associated with it.
+ * @param req - The request object.
+ * @param res - the response object
+ * @returns a message notifying the promotion's deletion if successfull, else an error message describing
+ * the error.
+ */
+const destroy = async (req, res) => {
+  const promotionId = req.params.promotionId;
+  try {
+    const Promotion = await promotion.findByPk(promotionId);
+    if (typeof Promotion === 'undefined' || Promotion === null)
+      return responses.errorDTOResponse(res, 404, 'Promocion no encontrada');
+
+    const { img: fileurl } = Promotion;
```

Comment:
```suggestion
    const promotion = await Promotion.findByPk(promotionId);
    if (typeof promotion === 'undefined' || promotion === null)
      return responses.errorDTOResponse(res, 404, 'Promocion no encontrada');

    const { img: fileurl } = promotion;
```
Porfavor... enserio porfavor ;u;

Reply:
Ño e tu madre

Reply:
Si explota es tu culpa, njd, corregir otra vaina más

Reply:
Ya la comiteo y resuelvo luego si explota